AmethystAPI/src/minecraft/src/common/world/level/storage/GameRules.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <amethyst/Memory.hpp>
+
+class GameRule {
+public:
+    /* this + 0 */ std::byte padding0[8];
+    /* this + 8 */ std::string name;
+    /* this + 40 */ std::byte padding1[3];
```

Comment:
Ah sorry, should've also noticed that the padding was named wrong. This can be fine in some cases, but with things like multi inheritance this is horrendous to maintain. We typically use the offset as the number, so this would be padding40, and that way it never runs into conflicts with multi inheritance

Reply:
By the way, you can just push again to your fork to add a commit. You don't need to close and make a new pr

Reply:
alright, ill fix the padding rq

Reply:
done